lib/routes/seagullApi.js

```diff
@@ -48,6 +49,106 @@ module.exports = function (crudHandler, userApiClient, metrics) {
     });
   }
 
+  function stringToBoolean(value) {
+    return _.includes(['true', 'yes', 'y', '1'], _.trim(value).toLowerCase());
+  }
+
+  function parseUsersQuery(req) {
+    var query = {};
+
+    var trustorPermissions = _.trim(req.params.trustorPermissions);
```

Comment:
We do, but they are different concepts.  The custodial/custodian relationship refers to an account that can *manage* almost everything on another account (email, sharing, etc.). So, "custodian" is just a permission much like "view" or "upload".

The trustor/trustee relationship is about *who* is involved in the permissions.  The trustor is the user who is trusting the trustee to do something (this could be view, upload, or even custodian). So, a trustor is trusting while the trustee is being trusted.  

Both the custodian/custodial and trustor/trustee terms are from the financial/banking world and have very specific definitions with little ambiguity.  One of the reasons I chose them.  They certainly aren't perfect, but the best I could come up with.  Please let me know if you have anything better.